### ISSUE TITLE
See list of tx involving this object

### ISSUE THREAD

**Issue by bwann52** [MystenLabs/sui#1210]
While on NFTs page, see list of transactions involving this NFT/object. Helps users gauge the value of these NFTs and become curious to click through to project url. Furthermore as owner they can easily track thru history and see activity.

**Comment by todd-mystenlabs** [comment]
@bwann52 - how does this differ from #1211?

**Comment by bwann52** [comment]
Yes, this one is about showing all txs. Whereas #1211 will require some UX/UI design.
